packages/react/src/components/MenuBar/index.test.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import axe from '../../axe';
+import MenuBar from '../MenuBar';
+import { TopBarItem } from '../TopBar';
+import * as viewportUtils from '../../utils/viewport';
+
+const [right, left] = [39, 37];
+const child = (
+  <TopBarItem>
+    <div />
+  </TopBarItem>
+);
+
+test('renders', () => {
+  render(<MenuBar>{child}</MenuBar>);
+  expect(screen.getByRole('menubar')).toBeTruthy();
+});
+
+test('supports falsy children', () => {
+  const { container } = render(
+    <MenuBar>
+      {child}
+      {false && child}
+    </MenuBar>
+  );
+  expect(container).toBeTruthy();
+  expect(container.querySelectorAll('div').length).toBe(1);
+});
+
+test('handles left arrow', async () => {
+  render(
+    <MenuBar>
+      <TopBarItem>test-1</TopBarItem>
+      <TopBarItem>test-2</TopBarItem>
+      <TopBarItem>test-3</TopBarItem>
+    </MenuBar>
+  );
+
+  const menuItems = screen.getAllByRole('menuitem');
```

Comment:
We should be firing events against menubar, not menuitems.

```suggestion
  const menuItems = screen.getAllByRole('menuitem')
  const menubar = screen.getByRole('menubar');
```

Access to both might still be useful, so you can validate that the right menuitem was activated.

Reply:
Found this issue, but my case is 100% opposite

Reply:
And also, found, that we do not handle `keyDownEvent` on the `menuBar` component, we pass this handler to the children. So I believe that we should test `keyDown` handler in menuItem although

If we try to trigger `onKeyDown` on `MenuBar` it has not even triggered

Reply:
Regarding your first comment on focusing `menuBar`, that's expected because menu bar isn't focusable.

For the latter comment, you're right - we do handle it on the child elements, I just didn't read the code fully. 🤦